lib/screens/create_region_screens/review_region_information.dart

```diff
@@ -4,20 +4,41 @@ import 'package:seeds/components/flat_button_long.dart';
 import 'package:seeds/components/full_page_error_indicator.dart';
 import 'package:seeds/components/full_page_loading_indicator.dart';
 import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/screens/create_region_screens/components/create_region_confirmation_dialog.dart';
+import 'package:seeds/screens/create_region_screens/components/review_region_header.dart';
 import 'package:seeds/screens/create_region_screens/interactor/viewmodels/create_region_bloc.dart';
-import 'package:seeds/screens/create_region_screens/review_region_information/components/review_region_header.dart';
+import 'package:seeds/screens/create_region_screens/interactor/viewmodels/create_region_page_commands.dart';
+
 import 'package:seeds/utils/build_context_extension.dart';
 
 class ReviewRegion extends StatelessWidget {
   const ReviewRegion({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return BlocBuilder<CreateRegionBloc, CreateRegionState>(
+    return BlocConsumer<CreateRegionBloc, CreateRegionState>(
+      listenWhen: (_, current) => current.pageCommand != null,
+      listener: (context, state) {
+        final pageCommand = state.pageCommand;
+
+        if (pageCommand is ShowCreateRegionConfirmation) {
+          FocusScope.of(context).unfocus();
+          showDialog<void>(
+            context: context,
+            barrierDismissible: false,
+            builder: (_) {
+              return BlocProvider.value(
+                value: BlocProvider.of<CreateRegionBloc>(context),
+                child: const CreateRegionConfirmationDialog(),
+              );
+            },
+          );
+        }
+      },
       builder: (context, state) {
         switch (state.pageState) {
           case PageState.loading:
-            return const FullPageLoadingIndicator();
+            return const Scaffold(body: FullPageLoadingIndicator());
```

Comment:
no, We talked about this. 
This page Scaffold needs to be outside, then the subcomponents dont need to be inside an Scaffold

Reply:
Yes but in this case the app Bar is different, the fullPageLoading doest need it, if the user goes back while Region Creation is loading will break things.

Reply:
what will break?